src/open_inwoner/configurations/admin.py

```diff
@@ -60,6 +60,9 @@ def delete_model(self, request, obj):
         else:
             super().delete_model(request, obj)
 
+    class Media:
+        css = {"all": ("css/admin/admin_overrides.css",)}
```

Comment:
I'm not sure why (as it was working before) but when I upload any kind of TTF file, it does correctly end up in the Media directory, but I do not see anything happening on the front-end (perhaps we need a fallback font somewhere?).
Here you can download clearly different TTF fonts for testing: https://www.fontsquirrel.com/fonts/acme

Reply:
The font faces definitions were missing the custom fonts after rebasing. Also, I had to specify the font-family for our `utrecht-heading`. Should work now.